cg/models/orders/validators/excel_sample_validators.py

```diff
@@ -33,12 +33,17 @@ def validate_data_analysis(data_analysis):
 
 
 def numeric_value(value: Optional[str]):
```

Comment:
can we get a return type hint?

Reply:
So the function basically checks whether the given value is a valid float or int? It does not perform any type conversion?

Reply:
Yeah, I do not intend to change any output types in this PR, I just wanted to patch some potential bypasses. So it will still return an Optional[str]. I'll add a type hint